docs-rtd/explanation/channel-revision.md

```diff
@@ -0,0 +1,45 @@
+(channel-revision)=
+# Channel and Revision
+
+The `channel` and `revision` fields can be set in your Terraform plan when deploying a Juju application.
+
+Example:
+
+```terraform
+resource "juju_application" "this" {
+    model = <model>
+
+    charm {
+        name     = "<charm-name>"
+        channel  = "<channel-name>"
+        revision = "<revision-number>"
+    }
+}
+```
+
+Both fields are optional. The following scenarios describe what happens when they are omitted:
```

Comment:
This says the scenarios describe what happens when they are omitted, but the first example is both set?